rpi_backlight.py

```diff
@@ -45,38 +46,53 @@ def _set_value(name: str, value: Any) -> None:
             _perm_denied()
 
 
+def set_brightness_value(value: Any) -> None:
```

Comment:
`value: Any` isn't useful at all, and please remove this function as well - there's already `set_brightness`, so there's no value in having another, similar function!

Reply:
It is a helper method which reduced the number of lines of code by a lot

Reply:
I changed it to int though
Check out what that function does though!

Reply:
First of all: if it's not part of the public API, name it with a leading underscore as per convention. Secondly I still don't see its value, as 2 of 4 usages of `set_brightness_value(value)` can just be replaced by `_set_value("tinker_mcu_bl", value)`.